app/redux/selectors/SetsSelectors.js

```diff
@@ -367,3 +367,37 @@ export const getSlowestDurationEver = (state, set) => {
 };
 
 export const getRevision = (state) => stateRoot(state).revision;
+
+// 1rm
+
+export const getExerciseVelocities = (state, exercise) => {   
+    const historySets = getHistorySets(state);
+    const workoutSets = getWorkoutSets(state);
+
+    const sets = historySets.concat(workoutSets);
+    let data = [];
+
+    sets.forEach((set) => {
+        if (set.exercise === exercise) {
+            data.push([set.weight, set.reps[0].data[2]]);
```

Comment:
set.reps[0].data[2]] is really complex

- [ ] use data map if possible instead? If we launch a v4 and it doesn't have velocity in the same spot, that's an issue